history-of-the-quakers-v4/original/04-book-6-chapter-17.adoc

```diff
@@ -132,14 +147,24 @@ and shall desire my brother to inform me, wherein I can be most serviceable to t
 and advise you what passeth,
 so conclude with my dear love and salutation to you and friends in general.
 
+[.signed-section-closing]
 I remain,
 
+[.signed-section-signature]
 Jacob Hagen.
 
-A Letter from Friends at Frederickstadt.
+--
+
+[.embedded-content-document.letter]
+--
+
+[.blurb]
+=== A Letter from Friends at Frederickstadt.
```

Comment:
Let's take a tiny bit of liberty here and change this to be a combined single letter-heading, like so:

```ASCIIDOC
[.letter-heading]
A Letter from Friends at Frederickstadt to the Friends, Fathers and Elders at London.
```